lib/plugins/blocks.js

```diff
@@ -413,24 +440,26 @@ function inject (bot, { version }) {
   })
 
   bot.updateSign = (block, text) => {
-    const lines = text.split('\n')
+    const lines = [...text.split('\n'), '', '', '']
+      .slice(0, 4)
+      .map(JSON.stringify)
```

Comment:
this would pass extra arguments to JSON.stringify, im not sure what effect it would have so would it be better to `json => JSON.stringify(json)`?

Reply:
yeah